packages/types/src/coreExtensions.ts

```diff
@@ -8,3 +8,4 @@ export * as AppPanels from "./coreExtensions/appPanels";
 export * as Commands from "./coreExtensions/commands";
 export * as ComponentEditor from "./coreExtensions/componentEditor";
 export * as Common from "./coreExtensions/common";
+export * as LinkRedirect from "./coreExtensions/linkRedirect";
```

Comment:
You forgot to register the type for the Webpack require function or module declarations: https://moonlight-mod.github.io/dev/core-extensions/#adding-types